paddle/fluid/framework/tensor_impl.h

```diff
@@ -162,6 +162,37 @@ inline Tensor& Tensor::ShareDataWith(const Tensor& src) {
   return *this;
 }
 
+inline Tensor& Tensor::ShareDataWith(const Tensor* src, size_t offset) {
```

Comment:
Can we use `Tensor::Slice` instead of this new function?

Reply:
Done.